covidfaq/config.py

```diff
@@ -6,6 +6,8 @@
 class Config(BaseSettings):
     elastic_search_host: str = "faq-master"
     elastic_search_port: int = 9200
+    elastic_search_use_ssl: bool = True
```

Comment:
```suggestion
    elastic_search_use_ssl: bool = False
```
Because the communication intra-cluster is http and not https